scripts/scil_compute_local_tracking.py

```diff
@@ -68,23 +97,37 @@ def _build_arg_parser():
     add_sphere_arg(track_g, symmetric_only=True)
     track_g.add_argument('--sub_sphere',
                          type=int, default=0,
-                         help='Subdivides each face of the sphere into 4^s new faces. '
-                              '[%(default)s]')
+                         help='Subdivides each face of the sphere into 4^s new'
+                              ' faces. [%(default)s]')
     add_seeding_options(p)
+
+    gpu_g = p.add_argument_group('GPU options')
+    gpu_g.add_argument('--use_gpu', action='store_true',
```

Comment:
This should be outside of the group, I think

Reply:
I don't know, this is the most elementary gpu option, no?

Reply:
For me, it does go in GPU options, but other GPU options should be things like, I don't know, max memory, rng seed for torch, etc. I think that batch size does belong here too.


Then, forward only and sh_interp are *not* GPU options. They are options that could one day be available in the CPU tracking too. They should go in something like "Options only available with GPU for now". Or in normal options, but with an explicit help message stating that this is not available on CPU yet.